README.md

```diff
@@ -849,6 +850,21 @@ this._viewer.getPageCropBox(1).then((cropBox) => {
 });
 ```
 
+##### setCurrentPage
+Set current page of the document.
+
+Parameters:
+
+Name | Type | Description
+--- | --- | ---
+pageNumber | integer | the page number for the target crop box. It is 1-indexed
+
+Return a Promise (will be rejected if provided page number is out of bound).
```

Comment:
i thought we agreed on resolve with boolean?

Reply:
Yeah did not change that yet, sorry